apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -180,6 +181,12 @@ export class PublishAction extends BaseRushAction {
       parameterShortName: undefined,
       description: 'If this flag is specified with --publish, packages will be published with --force on npm'
     });
+    this._applyGitTagsOnPack = this.defineFlagParameter({
```

Comment:
We should add a check to `run` to cause an error if `--publish` and `--pack` aren't also specified.

Reply:
Shouldn't we only require `--pack`? That way it will spit out a dry run of the tags it's going to add.

Reply:
Ah yeah good point.